src/routes/evalutation/route.ts

```diff
@@ -0,0 +1,182 @@
+import { z, createRoute } from "@hono/zod-openapi";
+import { EvaluationSchema } from "../../schemas/evaluation.js";
+
+export const createEvaluation = createRoute({
+  method: "put",
+  path: "/",
+  request: {
+    body: {
+      content: {
+        "application/json": {
+          schema: z.object({
+            projectId: z.string().uuid().openapi({
+              description:
+                "UUID of the project for which the evaluation is created",
+              example: "f47c9e99-10a9-4c12-8a8b-2f14ef9a9cba",
+            }),
+            score: z.number().min(0).max(10).openapi({
+              description: "Score for the evaluation, between 0 and 10",
+              example: 8,
+            }),
+          }),
+        },
+      },
+    },
+  },
+  responses: {
+    201: {
+      description: "Evaluation created successfully",
+      content: {
+        "application/json": {
+          schema: z.object({
+            id: z.string().uuid().openapi({
+              example: "9bdb42fd-5f35-4c0a-9bcb-9d2f013df01f",
+            }),
+            projectId: z.string().uuid(),
+            score: z.number(),
+          }),
+        },
+      },
+    },
+    403: {
+      description: "Unauthorized - User does not have permission",
+    },
+    404: {
+      description: "Project not found",
+    },
+    500: {
+      description: "Server error",
+    },
+  },
+});
+
+export const getEvaluation = createRoute({
+  method: "get",
+  path: "/",
+  responses: {
+    200: {
+      description: "List of evaluations",
+      content: {
+        "application/json": {
+          schema: z.array(
+            z.object({
+              id: z.string().uuid(),
+              createdAt: z.string(),
+              updatedAt: z.string(),
+              projectId: z.string().uuid(),
+              score: z.number(),
+            })
+          ),
+          example: [
```

Comment:
dont do it like this, use the `.openapi({example: })` method, this leads to inconsistencies